scripts/setup-linux.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Add source for git 
+sudo add-apt-repository ppa:git-core/ppa -y
+
+# Add source for unity hub
+wget -qO - https://hub.unity3d.com/linux/keys/public | gpg --dearmor | sudo tee /usr/share/keyrings/Unity_Technologies_ApS.gpg > /dev/null
+sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/Unity_Technologies_ApS.gpg] https://hub.unity3d.com/linux/repos/deb stable main" > /etc/apt/sources.list.d/unityhub.list'
+
+# Update apt-get 
+sudo apt-get update
+
+# Install prereqs
+sudo dnf install openssl1.1 openssl-libs
```

Comment:
Isn't dnf a redhat thing, while the rest of these are debian based package installer things?

Reply:
Interesting - this is an artifact of me putting together several script examples I found. I think this line can be removed though. I'll test it and make the change.

Reply:
Confirmed that that line in the script is superfluous.